### ISSUE TITLE
Set path for "github-copilot" folder

### ISSUE THREAD

**Issue by rjorge99** [zbirenbaum/copilot.lua#39]
Hi, is this running in windows? I have not been able to make it work, and I was wondering if the file host.json created by copilot in the setup process, is always looked for in "~/.config/github-copilot".

In windows is located at "C:\Users\user-name\AppData\Local\github-copilot".

**Comment by zbirenbaum** [comment]
This is handled by the copilot LSP, the plugin doesn't have to worry about it. As long as it's in the same place as where copilot.vim put it, it shouldn't matter at all. 

If it's not working, please provide the output of :LspLog :LspInfo and :CmpStatus

**Comment by zbirenbaum** [comment]
Are you using copilot-cmp? Copilot should not be an unavailable source.

**Comment by rjorge99** [comment]
Hi, yep, It's installed.. these are some of the things installed with packer..

`                    packer.nvim - Total plugins: 43
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 • Comment.nvim
 • DAPInstall.nvim
 • LuaSnip
 • alpha-nvim
 • bufferline.nvim
 • cmp-buffer
 • cmp-nvim-lsp
 • cmp-nvim-lua
 • cmp-path
 • cmp_luasnip
 • copilot-cmp
 • copilot.lua
 • darkplus.nvim
`

**Comment by zbirenbaum** [comment]
So I started having a similarish issue, where completions took a really really long time to show up. It may be that you didn't see any because of that, as I had to wait like 15-20 seconds to get a result. This is on Microsoft's end unfortunately, and is likely due to a parameter change in the panel solutions call. I have reverted the default to be getCompletionsCycling for now, as it still appears quickly, although that does mean that recommendations will be limited to 1-3 entries.

Try updating and ensure you are using getCompletionsCycling and let me know what happens.
